multi-agent/adapters/outbound/mongo/scheduled_prompt_repository.py

```
@@ -0,0 +1,130 @@
+    def save(self, prompt: ScheduledPrompt) -> str:
+        now = datetime.now(timezone.utc)
+        doc = prompt.model_dump(mode="json")
+        doc["created_at"] = now
+        doc["updated_at"] = now
+        self._col.insert_one(doc)
+        return prompt.id
+
+    def load(self, prompt_id: str) -> ScheduledPrompt:
+        doc = self._col.find_one({"id": prompt_id})
+        if not doc:
+            raise KeyError(f"No scheduled prompt with id={prompt_id}")
```

Ruff (0.15.21)

[warning] Avoid specifying long messages outside the exception class

(TRY003)